packages/bot-web-ui/package.json

```diff
@@ -108,4 +108,4 @@
         "html-react-parser": "5.1.12",
         "yup": "^0.32.11"
     }
-}
+}
```

Comment:
can we revert this linter change ?

Reply:
sure, once I'll update the version of the `@deriv-com/quill-ui` and add a new component

Reply:
done